docker-compose/docker-compose.yaml

```diff
@@ -1,5 +1,53 @@
 version: '2.4'
 services:
+  # Description: This container will migrate the Postgres `frontend`,
+  # `codeintel`, and `codeinsights` databases
+  #
+  # Disk: None
+  # Ports: None
+  #
+  # This container should start after the database instance starts, perform the migrations
+  # and then exit cleanly. After exiting the `frontend` service is allowed to start.
+  #
+  # If you're using a DB instance hosted outside of docker-compose, the environment variables
+  # for this container will need to be updated to reflect the new connection information.
+  migrator:
+    container_name: migrator
+    image: 'index.docker.io/sourcegraph/migrator:122149_2021-12-17_1f7179c'
```

Comment:
This new image should be added to the [list of images here](https://github.com/sourcegraph/sourcegraph/blob/main/enterprise/dev/ci/images/images.go#L58) so it can get auto-updated by Renovate and the release tooling.